### PR TITLE
Fixes in 1.3.1 GC rules with updated doc links and completing control response

### PR DESCRIPTION
**Pull request by rutvik23**
#### Description:
- Fixing `eviction_thresholds_set_hard_memory_available_master/rule.yml` to reflect correct ocil_clause
- Updating doc links to reflect the latest OCP pages
- Also adding the control response in control file cis_ocp_1_3_0 to reflect on status of the GC rules that should exist

#### Rationale:

- OCP4 CIS 1.3 cleanup and rule fixes

#### Review Hints:

- Please validate changes in `eviction_thresholds_set_hard_memory_available_master/rule.yml` and the control response file

### CODE REVIEW

File: controls/cis_ocp_1_3_0/section-1.yml [hunk 1]
```diff
@@ -329,8 +329,12 @@ controls:
     controls:
     - id: 1.3.1
       title: Ensure that garbage collection is configured as appropriate
-      status: partial
-      rules: [] 
+      status: Completed
```
Comment (Mab879) [inline controls/cis_ocp_1_3_0/section-1.yml:332]:
```
  RuntimeError: Unable to parse controls from /home/runner/work/content/content/controls/cis_ocp_1_3_0.yml: The given status 'Completed' in the control '1.3.1' was invalid. Please use one of the following: ['automated', 'documentation', 'does not meet', 'inherently met', 'manual', 'not applicable', 'partial', 'pending', 'planned', 'supported']
```
`Completed` is not supported. Please use one of the supported options.

**Review by Mab879 — DISMISSED** [review]
I have not review the Open Shift content, just the build errors.

File: controls/cis_ocp_1_3_0/section-1.yml (hunk 1 above)
Comment (jhrozek) [reply controls/cis_ocp_1_3_0/section-1.yml:332]: @rutvik23 is this one of the controls that are going away? In that case I would set the status as pending for now and add a note that this one is going away

Comment (rutvik23) [reply controls/cis_ocp_1_3_0/section-1.yml:332]: @jhrozek yes, (if i recall correctly) few rules in this control are going away but the rules with  _master _worker are going to stay. As per new CIS revision, we are also moving this rule into different control section altogether, hence keeping this in `Pending` state to be re-considered for future developments.

**Comment by jhrozek** [comment]
Looks good! I'm going to kick the CI tests that just appeared to have issues provisioning the environment and then we can merge.

**Comment by jhrozek** [comment]
The two remaining CI failures are not related to this PR. Merging.

**Pull request merged**